### DISCUSSION TITLE
Hello is there any way i can stop the font-size inherit & font-weight inherit from just one file which is navbar.tsx (using next.js)

### DISCUSSION THREAD

**Discussion by Srinivasan-Akash** [tailwindlabs/tailwindcss#12813] [Help]
Hello is there any way i can stop the font-size inherit & font-weight inherit from just one file which is navbar.tsx (using next.js)

**Comment by wongjn** [comment]
You could apply `font-size` and `font-weight` to override the default `inherit` values of the HTML elements rendered by `navbar.tsx`.

**Reply by Srinivasan-Akash** [reply]
Yes, i have tried that even with !important tag

**Reply by wongjn** [reply]
Seems like you may need to increase the specificity of the CSS selector(s) to override whatever rule that is applying the `inherit` values.
